app-i18n/transifex-client/transifex-client-1.0.0.ebuild

```
@@ -0,0 +1,229 @@
+	"golang.org/x/net v0.0.0-20210326060303-6b1517762897/go.mod"
+	"golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be/go.mod"
+	"golang.org/x/oauth2 v0.0.0-20181106182150-f42d05182288"
+	"golang.org/x/oauth2 v0.0.0-20181106182150-f42d05182288/go.mod"
+	"golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f/go.mod"
+	"golang.org/x/sys v0.0.0-20180905080454-ebe1bf3edb33/go.mod"
+	"golang.org/x/sys v0.0.0-20180909124046-d0be0721c37e/go.mod"
+	"golang.org/x/sys v0.0.0-20181122145206-62eef0e2fa9b/go.mod"
+	"golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod"
+	"golang.org/x/sys v0.0.0-20190507160741-ecd444e8653b/go.mod"
+	"golang.org/x/sys v0.0.0-20190916202348-b4ddaad3f8a3/go.mod"
+	"golang.org/x/sys v0.0.0-20191026070338-33540a1f6037/go.mod"
+	"golang.org/x/sys v0.0.0-20200302150141-5c8b2ff67527/go.mod"
+	"golang.org/x/sys v0.0.0-20201119102817-f84b799fce68/go.mod"
+	"golang.org/x/sys v0.0.0-20210320140829-1e4c9ba3b0c4/go.mod"
+	"golang.org/x/sys v0.0.0-20210324051608-47abb6519492/go.mod"
+	"golang.org/x/sys v0.0.0-20210330210617-4fbd30eecc44"
+	"golang.org/x/sys v0.0.0-20210330210617-4fbd30eecc44/go.mod"
+	"golang.org/x/term v0.0.0-20201117132131-f5c789dd3221/go.mod"
+	"golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1/go.mod"
+	"golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d"
+	"golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d/go.mod"
+	"golang.org/x/text v0.3.0/go.mod"
+	"golang.org/x/text v0.3.3/go.mod"
+	"golang.org/x/text v0.3.5"
+	"golang.org/x/text v0.3.5/go.mod"
+	"golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod"
+	"golang.org/x/tools v0.0.0-20190328211700-ab21143f2384/go.mod"
+	"google.golang.org/appengine v1.1.0/go.mod"
+	"google.golang.org/appengine v1.3.0"
+	"google.golang.org/appengine v1.3.0/go.mod"
+	"gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c"
+	"gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c/go.mod"
+	"gopkg.in/fsnotify.v1 v1.4.7/go.mod"
+	"gopkg.in/ini.v1 v1.62.0"
+	"gopkg.in/ini.v1 v1.62.0/go.mod"
+	"gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7/go.mod"
+	"gopkg.in/warnings.v0 v0.1.2"
+	"gopkg.in/warnings.v0 v0.1.2/go.mod"
+	"gopkg.in/yaml.v2 v2.2.1/go.mod"
+	"gopkg.in/yaml.v2 v2.2.2/go.mod"
+	"gopkg.in/yaml.v2 v2.2.3/go.mod"
+	"gopkg.in/yaml.v2 v2.2.4/go.mod"
+	"gopkg.in/yaml.v2 v2.3.0"
+	"gopkg.in/yaml.v2 v2.3.0/go.mod"
+	"gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c/go.mod"
+	"gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b"
+	"gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b/go.mod"
+	)
+go-module_set_globals
+
+SRC_URI="https://github.com/transifex/cli/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	${EGO_SUM_SRC_URI}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+S=${WORKDIR}/cli-${PV}
```

```suggestion
S="${WORKDIR}"/cli-${PV}
```

.. and I usually put it directly under S (with no blank new line between them).